resources/test/fixtures/ruff/RUF005.py

```diff
@@ -0,0 +1,7 @@
+foo = [4, 5, 6]
+bar = [1, 2, 3] + foo
+quux = (7, 8, 9) + bar
+spam = quux + (10, 11, 12)
+eggs = spam + [13, 14, 15]
+# TODO: this results in very strange (buggy) results:
+#  chain = ['a', 'b', 'c'] + eggs + ('yes', 'no', 'pants') + quux
```

Comment:
I think we should resolve this prior to merging.

Maybe we avoid applying this if one of the expressions is itself a BinOp? So we'd flag `Consider `['a', 'b', 'c', *eggs]` instead of concatenation` but not `Consider `*['a', 'b', 'c'] + eggs, 'yes', 'no', 'pants'` instead of concatenation`.

Reply:
I added a guard for the splattee, which seems to clean things up.